internal/controller/driver_controller.go

```diff
@@ -981,7 +982,41 @@ func (r *driverReconcile) reconcileNodePluginDeamonSetForCsiAddons() error {
 
 	log := r.log.WithValues("csiAddonsDaemonSetName", daemonSet.Name)
 
-	if !ptr.Deref(r.driver.Spec.DeployCsiAddons, false) {
+	// FIXME: Spec.DeployCsiAddons defaults to true, causes CSI-Addons for CephFS enabled by default
+	withCsiAddonsDaemonSet := false // ptr.Deref(r.driver.Spec.DeployCsiAddons, false)
```

Comment:
@nixpanic need to remove the FixME and the comment as no more defaulting to true

Reply:
```go
ptr.Deref(r.driver.Spec.DeployCsiAddons, false)
```
returns `true` even when `.Spec.DeployCsiAddons` is not set in the CR? I think that is still the case, or at least I see that in my test environment.